### PR TITLE
Updating to include en-* languages

### PR DESCRIPTION
**Pull request by sammiili**
Updated to include en-* languages in PTT and Voice activation

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/cortana-in-teams.md [hunk 1]
```diff
@@ -93,11 +93,11 @@ At the device level, you can configure Cortana to be used in two different ways.
 - by tapping on a microphone, which is called Cortana _Push to talk_
 - by saying "Hey, Cortana", which is called _Cortana Voice Activation_
 
-Cortana _Push to talk_ is enabled by default if your device is set up to use locale en-us. [Learn More.](/MicrosoftTeams/rooms/console#to-apply-your-desired-language)  Cortana icon will displace the _Present_ button under the _More…_ menu in your Teams Room console. To disable Cortana _Push to talk_ use PowerShell.[Learn More.](/powershell/module/skype/new-csteamscortanapolicy?view=skype-ps#example-1)
+Cortana _Push to talk_ is enabled by default if your room is set with any of the following languages: en-au (Australia), en-ca (Canada), en-gb (United Kingdom), en-in (India), en-us (United States). [Learn More.](/MicrosoftTeams/rooms/console#to-apply-your-desired-language)  Cortana icon will displace the _Present_ button under the _More…_ menu in your Teams Room console. To disable Cortana _Push to talk_ use PowerShell.[Learn More.](/powershell/module/skype/new-csteamscortanapolicy?view=skype-ps#example-1)
```
Comment (JohanFreelancer9) [inline Teams/cortana-in-teams.md:96]:
```suggestion
Cortana _Push to talk_ is enabled by default if your room is set up with any of the following languages: en-au (Australia), en-ca (Canada), en-gb (United Kingdom), en-in (India), en-us (United States). [Learn More.](/MicrosoftTeams/rooms/console#to-apply-your-desired-language) Cortana icon will displace the _Present_ button under the _More…_ menu in your Teams Room console. To disable Cortana _Push to talk_ use PowerShell.[Learn More.](/powershell/module/skype/new-csteamscortanapolicy?view=skype-ps#example-1)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @sammiili, kindly commit the minor suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@HowlinWolf-92 PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**